src/quickdocumentdialog.cpp

```diff
@@ -303,15 +282,65 @@ void QuickDocumentDialog::Init()
         item->setCheckState(Qt::Unchecked);
     }
 
+	QTableWidget *table = ui.tableWidgetPackages;
+	table->clear();
+	table->setColumnCount(2);
+	table->setHorizontalHeaderLabels( QStringList( { tr("Package"), tr("Description") } ) );
+	table->horizontalHeader()->setStretchLastSection(true);
+	table->setSelectionMode(QAbstractItemView::ExtendedSelection);
+	table->setSelectionBehavior(QAbstractItemView::SelectRows);
+	table->verticalHeader()->hide();
+
+	//each QStringList holds 2 items: the name of the package, and a short package description. These constitute a row of the table of the packages tab
+	QList<QStringList> packages = QList<QStringList>()
+		<< QStringList( {"amssymb"     , "Mathematical symbols from AMS"} )
+		<< QStringList( {"graphicx"    , "Graphics package, easily include images (s. Insert Graphic Wizard)"} )
+		<< QStringList( {"hyperref"    , "Support for hyperlinks in your document"} )
+		<< QStringList( {"mathtools"   , "Extension package to amsmath incl. fixes for bugs in amsmath, loads amsmath"} )
+		<< QStringList( {"amsthm"      , "Define your theorem like env., has to be loaded after amsmath"} )
+		<< QStringList( {"nameref"     , "Reference to names of chapters, sections, ..., loaded by hyperref"} )
+		<< QStringList( {"thmtools"    , "Extention package to amsthm"} )
+		<< QStringList( {"xcolor"      , "Sophisticated package for colors, with table option use colors in tables"} )
+		;
+	//add user given packages
+	for (const QString& package:otherPackagesList){
+		packages << QStringList( { package, "" } );
+	}
+	//setup packages table 
+	int row=-1;
+	int tableHeight = table->horizontalHeader()->height() + 2;
+	for (const QStringList& data:packages){
+		++row;
+		QString pkgName = data[0],
+				pkgDescription = data[1];
+
+		QTableWidgetItem *itemPkgName = new QTableWidgetItem(pkgName);
+		QTableWidgetItem *itemPkgDescription = new QTableWidgetItem(pkgDescription);
+
+		itemPkgName->setFlags(Qt::ItemIsEnabled | Qt::ItemIsSelectable | Qt::ItemIsUserCheckable);
+		itemPkgDescription->setFlags(Qt::ItemIsEnabled | Qt::ItemIsSelectable);
+
+		table->setRowCount(row+1);
```

Comment:
probably not the most efficient way to build a table, but in this context it does not matter.

Reply:
What's the point? Always setting RowCount?

Reply:
yes, especially as you know the number from the start.